src/main/java/uk/gov/hmcts/reform/sscs/mappers/ServiceHearingValuesMapper.java

```diff
@@ -0,0 +1,118 @@
+                .caseAdditionalSecurityFlag(HearingsCaseMapping.shouldBeAdditionalSecurityFlag(caseData))
+                .facilitiesRequired(HearingsDetailsMapping.getFacilitiesRequired(caseData))
+                .listingComments(HearingsDetailsMapping.getListingComments(caseData.getAppeal(), caseData.getOtherParties()))
+                .hearingRequester(HearingsDetailsMapping.getHearingRequester())
+                .privateHearingRequiredFlag(HearingsDetailsMapping.isPrivateHearingRequired())
+                .leadJudgeContractType(HearingsDetailsMapping.getLeadJudgeContractType()) // TODO ref data isn't available yet. List Assist may handle this value
+                .judiciary(getJudiciary(caseDetails, referenceData))
+                .hearingIsLinkedFlag(HearingsDetailsMapping.isCaseLinked(caseData))
+                .parties(ServiceHearingPartiesMapping.buildServiceHearingPartiesDetails(caseData))
+                .caseFlags(PartyFlagsMapping.getCaseFlags(caseData))
+                .screenFlow(null)
+                .vocabulary(null)
+            .build();
+    }
+
+    public static Judiciary getJudiciary(SscsCaseDetails caseDetails, ReferenceData referenceData) {
+        SscsCaseData sscsCaseData = caseDetails.getData();
+        return Judiciary.builder()
+                .roleType(HearingsDetailsMapping.getRoleTypes())
+                .authorisationTypes(HearingsDetailsMapping.getAuthorisationTypes())
+                .authorisationSubType(HearingsDetailsMapping.getAuthorisationSubTypes())
+                .judiciarySpecialisms(HearingsDetailsMapping.getPanelSpecialisms(sscsCaseData, getSessionCaseCode(sscsCaseData, referenceData)))
+                .judiciaryPreferences(getPanelPreferences())
+                .build();
+    }
+
+    public static List<PanelPreference> getPanelPreferences() {
+        //TODO Need to retrieve PanelPreferences from caseData and/or ReferenceData
+        return new ArrayList<>();
+    }
+
+    public static HearingWindow getHearingWindow(SscsCaseData caseData) {
+        String hearingWindowStart = null;
+        if (Objects.nonNull(caseData.getEvents())) {
+            Event dwpResponded = caseData.getEvents().stream()
+                    .filter(c -> EventType.DWP_RESPOND.equals(c.getValue().getEventType()))
+                    .findFirst().orElse(null);
+
+            ZonedDateTime dwpResponseDateTime = Optional.ofNullable(dwpResponded)
+                    .map(Event::getValue)
+                    .map(EventDetails::getDateTime)
+                    .orElse(null);
+
+            if (Objects.nonNull(dwpResponseDateTime)) {
+                if (YesNo.isYes(caseData.getUrgentCase())) {
+                    hearingWindowStart = dwpResponseDateTime.plusDays(14).toLocalDate().toString();
+
+                } else {
+                    hearingWindowStart = dwpResponseDateTime.plusDays(28).toLocalDate().toString();
+                }
+            }
```

Comment:
Can you copy over the method to your PR and use that for the startDate?
https://github.com/hmcts/sscs-hearings-api/commit/0a24b7b3e782a6da2fccd029d53768cc2af1376b#diff-e7c6a67d83ad0f2c0339d7d50de210c6008fa931075eec7ababa9f423195c719R96-R11

Reply:
Address this for MVP

Reply:
Am adding new methods isCaseUrgent(@Valid SscsCaseData caseData) and getHearingWindowStart(@Valid SscsCaseData caseData, boolean autoListed) to my version of HearingDetailsMapping and then using them in ServiceHearingValuesMapper.
This means I'm implementing part of your changes, Lucas, so you may need to revert if this PR gets to master first.